.gitignore

```diff
@@ -1,11 +1,10 @@
 /.bundle/
 /.yardoc
-/Gemfile.lock
 /_yardoc/
 /coverage/
 /doc/
 /pkg/
 /spec/reports/
 /tmp/
-.ruby-*
 *.sw*
+.idea
```

Comment:
Could you add this to your global gitignore instead? (not blocking, but would be nice instead of personalized gitignores)

Reply:
I do have this in my global file, but many people don't, hence I'm used to adding it, to not end up with an .idea folder being included some day.